mreg_cli/commands/policy.py

```diff
@@ -366,14 +319,8 @@ def list_members(args: argparse.Namespace) -> None:
 
     :param args: argparse.Namespace (name)
     """
-    info = get_role(args.name)
-    manager = OutputManager()
-    if info["atoms"]:
-        manager.add_line("Name:")
-        for i in info["atoms"]:
-            manager.add_line(f" {i['name']}")
-    else:
-        manager.add_line("No atom members")
+    role = Role.get_by_name_or_raise(args.name)
+    role.output_atoms()
```

Comment:
Should we delegate all printing to the model like this, or does it clutter the model definition?

Reply:
I vastly prefer to delegate the printing. I'd like the managing of output and formatting to be found in as few places / files as possible.